jw3/example-daffodil-vscode
# Daffodil Packaging

Removed files dealing with the sampleWeb extension since it will not be needed. Updated the extension to check if the version daffodil-debugger the user enters is available, if it is not it will be downloaded, once the debugger files exist the debugger will be ran in the terminal.

How to test extension:
```
yarn install
yarn vscode:prepublish
yarn run vsce package
code --install-extension daffodil-debug-0.0.1.vsix
```
After those have been ran open up the `sampleWorkspace` folder without laucnhing it through vscode.
Then run `Debug test.md`, you should be prompted to enter debugger version, enter in `0.0.2`
After a short time you should now see in the terminal that the debugger is running then shortly after the extension should connect to the debugger.
